webapp/src/main/java/org/apache/atlas/web/rest/EntityREST.java

```diff
@@ -899,15 +901,25 @@ public EntityMutationResponse createOrUpdate(AtlasEntitiesWithExtInfo entities,
     }
 
     public static void validateAttributeLength(final List<AtlasEntity> entities) throws AtlasBaseException {
-        //Predicate to check attribute value exceeding length
-        Predicate<Map.Entry<String, Object>> predicateOfAttributeLengthExceedingLimit = attribute ->
-                attribute.getValue() instanceof String && ((String) attribute.getValue()).length() > HUNDRED_THOUSAND;
+        List<String> errorMessages = new ArrayList<>();
 
         for (final AtlasEntity atlasEntity : entities) {
-            Set<String> attributeKeys = org.apache.commons.collections4.MapUtils.emptyIfNull(atlasEntity.getAttributes())
-                    .entrySet().stream().filter(predicateOfAttributeLengthExceedingLimit).map(Map.Entry::getKey).collect(Collectors.toSet());
-            if (!attributeKeys.isEmpty()) {
-                throw new AtlasBaseException("Attribute(s) " + String.join(",", attributeKeys) + " exceeds limit of "+HUNDRED_THOUSAND+" characters");
+            for (Map.Entry<String, Object> attribute : atlasEntity.getAttributes().entrySet()) {
+
+                if (attribute.getValue() instanceof String && ((String) attribute.getValue()).length() > HUNDRED_THOUSAND) {
```

Comment:
Typecasting to string directly may cause NPE, use String.valueOf

Reply:
@n5nk , we are already checking instanceof, it will not evaluate next condition in case false, means in case value is null